src/utils.ts

```diff
@@ -64,14 +61,16 @@ export class CliUtils {
         if (err) {
           reject(err);
         }
-        JSZip.loadAsync(data).then(
-          (zip) => {
-            resolve(zip.file("export.data").async("string"));
-          },
-          (reason) => {
-            reject(reason);
-          }
-        );
+        import("jszip").then((JSZip) => {
+          JSZip.loadAsync(data).then(
```

Comment:
This seems to not work, I think we need to use the `default` here.

```suggestion
          JSZip.default.loadAsync(data).then(
```